addon/components/polaris-labelled.js

```diff
@@ -87,4 +87,15 @@ export default Component.extend({
    * @private
    */
   helpTextId: computedHelpTextId('id').readOnly(),
+
+  /**
+   * Flag indicating whether to render the error component
+   *
+   * @type {Boolean}
+   * @private
+   */
+  shouldRenderError: computed('error', function() {
+    let { error } = this;
```

Comment:
I think we still want to use Ember getters to support older versions of Ember.....from what I remember I run into issues trying to do this before